docker/runtime/ruby-2.4/event-trigger/kubeless.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+MOD_NAME = ENV['MOD_NAME']
+FUNC_HANDLER = ENV['FUNC_HANDLER']
+
+TOPIC_NAME = ENV['TOPIC_NAME']
+KAFKA_SVC = ENV.fetch('KUBELESS_KAFKA_SVC', 'kafka')
+KAFKA_NAMESPACE = ENV.fetch('KUBELESS_KAFKA_NAMESPACE', 'kubeless')
+KAFKA_HOST = "#{KAFKA_SVC}.#{KAFKA_NAMESPACE}:9092";
+
+MOD_ROOT_PATH = ENV.fetch('MOD_ROOT_PATH', '/kubeless/')
+MOD_PATH = [ MOD_ROOT_PATH, MOD_NAME, '.rb' ]
+puts('Loading', MOD_PATH.join())
+
+begin
+  require 'sinatra'
+  require 'kafka'
+  require  MOD_PATH.join()
+  KubelessFunction = Object.const_get(FUNC_HANDLER)
```

Comment:
As mentioned in the description, this loads the function differently to the HTTP one. I want to update the HTTP one with this method so the FUNC_HANDLER is actually used.

There may actually be a better way to load the function in Ruby, I currently don't like that a class need to be created with a `run` class method.